network/src/noise_wrapper/handshake.rs

```diff
@@ -222,7 +222,7 @@ impl NoiseWrapper {
         }
 
         // if on a mutually authenticated network
-        if let Some(anti_replay_timestamps) = &anti_replay_timestamps {
+        if let Some(anti_replay_timestamps) = anti_replay_timestamps {
```

Comment:
it feels cleaner to me to have anti_replay_timestamps.clone() instead of borrowing a pointer

Reply:
yeah `&Arc` is usually an anti-pattern imo. i'll add a commit to refactor these into fields

Reply:
done!